lib/install/engine/interactive/wizard.go

```diff
@@ -77,6 +78,14 @@ func (r *Engine) execute(ctx context.Context, installer install.Interface, confi
 	if err != nil {
 		return trace.Wrap(err)
 	}
+
+	if e.config.App.Manifest.OpsCenterDisabled() {
```

Comment:
This is too late to check it here. Ideally, this validation should run in the client process before it gets to the service and I'd move it [here](https://github.com/gravitational/gravity/blob/349f0824c66d74450fcabb451e332a40099b320d/tool/gravity/cli/config.go#L377) where the manifest is also already available.